src/cast_core/src/RunCommand.cpp

```diff
@@ -257,7 +257,8 @@ void actor::RunCommand::run() {
 
 actor::RunCommand::RunCommand(ActorContext& context)
     : Actor(context),
-      _client{std::move(context.client())},
+      _client{std::move(
+          context.client(context.get("ClientName").maybe<std::string>().value_or("Default")))},
```

Comment:
LGTM.

(Note for future reference: the CRUD and RunCommand actors are duplicating the default arg of "Default" here. It's  unlikely for other existing actors to require a configurable client name; if new use cases come up, we can switch the arg to `std::optional` to replace the `value_or()` part.